packages/optimizer/lib/transformers/AmpBoilerplateTransformer.js

```diff
@@ -28,7 +28,7 @@ const V0_CSS_URL = AMP_CACHE_HOST + '/' + V0_CSS;
  * runtime version is specified, v0.css will be inlined.
  */
 class AmpBoilerplateTransformer {
-  constructor(fetch = OneBehindFetch.create()) {
+  constructor(config, fetch = OneBehindFetch.create()) {
```

Comment:
nit: I'm wondering if fetch should be a parameter in itself or another option in the config.

Reply:
Yes, that could make sense. We should keep it in mind.